### PR TITLE
Adding support to setLanguageCode to FirebaseAuth

### PR DESCRIPTION
**Pull request by ffeu**
related issue:
flutter/flutter#19525

### CODE REVIEW

File: packages/firebase_auth/lib/firebase_auth.dart [hunk 1]
```diff
@@ -411,6 +411,15 @@ class FirebaseAuth {
     return currentUser;
   }
 
+  /// Sets the user-facing language code for auth operations that can be
+  /// internationalized, such as [sendEmailVerification].
```
Comment (kroikie) [inline packages/firebase_auth/lib/firebase_auth.dart]: Please include the formatting requirements (IETF in BCP47) in this comment.

Eg: as in [Android reference docs](https://firebase.google.com/docs/reference/android/com/google/firebase/auth/FirebaseAuth.html#setLanguageCode(java.lang.String))

**Comment by kroikie** [comment]
@ffeu Thanks for the PR, this looks good. Could you please:

- update comments on dart side.
- Bump version number of plugin
- Update CHANGELOG with change.

File: packages/firebase_auth/lib/firebase_auth.dart (hunk 1 above)
Comment (ffeu) [reply packages/firebase_auth/lib/firebase_auth.dart]: done

**Comment by kroikie** [comment]
LGTM

**Comment by basketball-ico** [comment]
Hello, the email `updateEmail` sent is in English, after I set different laguage like this
```Dart
await FirebaseAuth.instance.setLanguageCode('es');
final user = await FirebaseAuth.instance.currentUser();
await user.updateEmail('example@gmail.com');
``` 
I am doing correct?
Thanks

**Comment by ffeu** [comment]
@basketball-ico 
Hi, the language code must be in IETF, try `setLanguageCode('es-419')` (latin america spanish) or `setLanguageCode('es-ES')`(spain spanish).

PS: the best place for questions like this is [stackoverflow](https://stackoverflow.com/questions/tagged/flutter).

**Comment by basketball-ico** [comment]
@ffeu Hi thanks I try this, but the `updateEmail` is send in english, the `resetPassword`  and `emailVeification` email are sent in the language that I set.

**Pull request merged**